traitsui/tree_node.py

```diff
@@ -219,7 +220,7 @@ def has_children(self, object):
     def get_children(self, object):
         """ Gets the object's children.
         """
-        return getattr(object, self.children)
+        return xgetattr(object, self.children)
```

Comment:
```suggestion
        return xgetattr(object, self.children, None)
```
The default return value of `xgetattr` is `Undefined` (see https://github.com/enthought/traits/blob/master/traits/trait_base.py#L345).  For backwards compatibility you need the default to be `None`.